assets/css/admin-sett-style.css

```diff
@@ -0,0 +1,55 @@
+
+  /* setting admin page */
+  .form-table th{
+    width: 100%!important;
```

Comment:
Why a lot of `!important`? Also, can't we do all this with tailwind's classes.

Reply:
i had solved the !important.